foreman/data_refinery_foreman/foreman/test_main.py

```diff
@@ -0,0 +1,274 @@
+from unittest.mock import patch
+from datetime import timedelta
+from django.utils import timezone
+from django.test import TestCase
+from data_refinery_foreman.foreman import main
+from data_refinery_models.models import (
+    SurveyJob,
+    Batch,
+    BatchStatuses,
+    DownloaderJob,
+    ProcessorJob
+)
+
+
+class SurveyTestCase(TestCase):
+    def setUp(self):
+        survey_job = SurveyJob(source_type="ARRAY_EXPRESS")
+        survey_job.save()
+        self.survey_job = survey_job
+
+    def tearDown(self):
+        Batch.objects.all().delete()
+        DownloaderJob.objects.all().delete()
+        ProcessorJob.objects.all().delete()
+
+    def insert_batch(self):
```

Comment:
Actually, it accesses the `self.survey_job` which is created in the setUp function. I can see how you missed it though.